python/tvm/relax/op/transform.py

```diff
@@ -0,0 +1,282 @@
+
+
+def squeeze(data, axis: Optional[Union[int, List[int]]] = None) -> Expr:
+    """Squeeze axes in the array.
+
+    Parameters
+    ----------
+    data : relax.Expr
+        The input data to the operator.
+
+    axis : Optional[Union[int, List[int]]
+        The set of axes to remove.
+        If axis = None, remove all axis of dimensions 1.
+        If any specified axis has dimension that does not equal 1, it is an error.
+
+    Returns
+    -------
+    result : relax.Expr
+        The squeezed result.
+    """
+    if isinstance(axis, int):
+        axis = [axis]
+    return _ffi_api.squeeze(data, axis)  # type: ignore
+
+
+def flatten(data: Expr) -> Expr:
+    """Flatten all the tensor dimensions into one.
+
+    Parameters
+    ----------
+    data : Expr
+        The input data to the operator.
+
+    Returns
+    -------
+    result : Expr
+        The flattened result.
+    """
+    return _ffi_api.flatten(data)  # type: ignore
+
+
+def cast(data: Expr, dtype: str) -> Expr:
+    """Cast input tensor to the given data type.
+
+    Parameters
+    ----------
+    data : relax.Expr
+        The input data to the operator.
+
+    dtype: str
+        The target data type
+
+    Returns
+    -------
+    result : relax.Expr
+        The casted result.
+    """
+    return _ffi_api.cast(data, dtype)  # type: ignore
+
+
+def full(
+    fill_value: Expr,
+    shape: Union[PrimExprLike, Tuple[PrimExprLike], Expr],
+    dtype: Optional[str] = None,
+) -> Expr:
+    """Fill array with scalar value.
+
+    Parameters
+    ----------
+    fill_value : relax.Expr
+        The value to fill. Must be a scalar tensor.
+
+    shape : Union[PrimExprLike, Tuple[PrimExprLike], Expr]
+        The shape of the created tensor.
+
+    dtype : Optional[str]
+        The data type of the created tensor.
+        If dtype is not given, it will by default use the dtype of fill_value.
+
+    Returns
+    -------
+    result : relax.Expr
+        The result tensor.
+    """
+    return _ffi_api.full(fill_value, shape, dtype)  # type: ignore
+
+
+def full_like(data: Expr, fill_value: Expr) -> Expr:
+    """Construct a tensor such that
+    - its shape and dtype is the same as the input data tensor's,
+    - its value is filled with the input scalar fill value.
+
+    Parameters
+    ----------
+    data : relax.Expr
+        The input tensor, which provides the shape and dtype.
+
+    fill_value : relax.Expr
+        The value to fill. Must be a scalar tensor.
+
+    Returns
+    -------
+    result : relax.Expr
+        The result tensor.
+    """
+    return _ffi_api.full_like(data, fill_value)  # type: ignore
+
+
+def ones(shape: Union[PrimExprLike, Tuple[PrimExprLike], Expr], dtype: str) -> Expr:
+    """Construct a tensor of all ones, with the input shape and dtype.
+
+    Parameters
+    ----------
+    shape : Union[PrimExprLike, Tuple[PrimExprLike], Expr]
+        The shape of the created tensor.
+
+    dtype : Optional[str]
```

Comment:
If the dtype is not given as default, the annotation should be `str`
```suggestion
    dtype : str
```